docs/index.md

```diff
@@ -11,25 +11,24 @@ Explore the Semgrep docs and join an amazing community of engineering and securi
 - [Getting started](getting-started.md)
 - [Writing rules](writing-rules/overview.md)
 - [Running rules](running-rules.md)
+- [Running Semgrep in CI](running-in-ci.md)
 - [Managing CI policy](managing-policy.md)
-- [Integrations](integrations.md)
 - [Experiments](experiments/overview.md)
 
 ---
 
 # Overview
 
-
 Semgrep is a fast, open-source, static analysis tool that excels at expressing code standards — without complicated queries — and surfacing bugs early at editor, commit, and CI time. Precise rules look like the code you’re searching; no more traversing abstract syntax trees or wrestling with regexes.
 
 Semgrep encompasses:
 
-* [Semgrep CLI](getting-started.md) - the [open-source](https://github.com/returntocorp/semgrep) command-line tool at the heart of everything
```

Comment:
Yes, sorry about that! I've turned off my autoformatter since then for this repo, but I overlooked this when committing. Can this one formatting change stay to save me the trouble of undoing it?

Reply:
ended up undoing autoformat changes 😇